Update _P073_7DGT.ino

papperone opened the pull request:
Adding new commands (7dst, 7dsd, 7dtext), new options to show 12h or 24h clock and fixing some bugs

papperone committed:
Update _P073_7DGT.ino

Adding new commands (7dst, 7dsd, 7dtext), new options to show 12h or 24h clock and fixing some bugs

papperone commented:
it's long time since my last PR but I've no clue why so many errors while the plugin compiles perfectly in my local environment :(

Grovkillen commented:
It is complaining about not declaring variables.

papperone commented:
well it's complaiing about a lot of things of which I did not modify any code since last release...
I can only see I forgot to add PROGMEM to the static binary arrays declaration but I'm clueless what all the other dozens of errors means!

TD-er commented:
At first glance it looks like a forgotten #endif or something like that.

I will take a look at it later today.

papperone commented:
ok thanks, I have looked  for those and I've only the #ifdef USES_P073 at the beginning and its #endif at the end

papperone commented:
any updates? I just would like where is my mistake before I make futher developing...

TD-er commented:
Not yet, was quite busy last few days.
I will now have a look.

TD-er committed:
[P073_7DGT] Removed old (duplicate) content

The old version was still present in the file, so I removed the duplicate (old) part

TD-er commented:
The last part of your file still had the old version of the file.
So I removed that part and pushed it.

papperone commented:
that's weird.... ok thanks for finding the mess I have done somehow! 🗡

TD-er committed:
[P073_7DGT] Fix uninitialized p073_dispdigit

TD-er commented:
Added an extra commit to fix compiler error about uninitialized variable `p073_dispdigit`

TD-er commented:
@papperone What is the status of this PR?

papperone commented:
sorry tested last night and it works, so fine for me to release...
I will need some time to work to implement handling of multiple display which still does not work!

Grovkillen commented:
Should we move this to normal? I have used it in one node for a long time and like it. It feels stable enough to be set to normal.

papperone commented:
well I've 2 display always on since long time... I think we can move to NORMAL unless some other users disagree...
I can always add the multi-display handling at later stage as I need to implent and test it still

Grovkillen commented:
If you commit an update removing the testing flag we'll merge it today.

papperone commented:
mmmm.... how can I add a second file to this PR as I need to update as well "define_plugin_sets.h" ?
or shoudl I do a second PR for this change?

Grovkillen commented:
If you update your branch it'll update this PR. It'll automatically know that `define_plugin_sets.h` was changed.

papperone committed:
Update _P073_7DGT.ino

papperone commented:
done... Travis is checking ;)

TD-er commented:
I merged it.
Would be nice if the titles of the PR's are a bit more descriptive.
Remember those will end up in the changelog.

Grovkillen commented:
It's the commit names that end up in the change log.

papperone commented:
shoudl I put the first post (which briefly describe the changes) somewhere to have it in the change log ?

Grovkillen commented:
Do it for future PRs. For this one is ok

TD-er commented:
I took some of that information and put it in the merge commit.

papperone commented:
thanks!
